81E00600.4D0200.dts

```diff
@@ -0,0 +1,132 @@
+
+/dts-v1/;
+
+/ {
+        name = "";
+        compatible = "ibm,opal-in-memory-counters";
+        #address-cells = <0x1>;
+        #size-cells = <0x1>;
+        ima-nest-offset = <0x320000>;
+        ima-nest-size = <0x30000>;
+        version-id = "";
+
+        mcs {
+                compatible = "ibm,ima-counters-nest";
```

Comment:
'unit' and 'scale' properties have been added to the root node itself. For 'mcs' the 'unit' and 'scale' are the same for all its event nodes.
Once we have a source where unit and scale information is provided per event, then we need to make changes further, to accomodate this per event information.